python/_tskitmodule.c

```diff
@@ -8136,6 +8136,26 @@ Tree_get_mrca(Tree *self, PyObject *args)
     return ret;
 }
 
+static PyObject *
+Tree_get_num_children(Tree *self, PyObject *args)
+{
+    PyObject *ret = NULL;
+    size_t num_children;
```

Comment:
could make this an unsigned int here and avoid the cast below.

Reply:
Good catch - copy-paste snafu. Done